tests/Directory.Build.props

```diff
@@ -8,6 +8,7 @@
     <IsPackable>false</IsPackable>
     <EnablePackageValidation>false</EnablePackageValidation>
     <AssemblyOriginatorKeyFile>../../build/MSAL.snk</AssemblyOriginatorKeyFile>
+    <SuppressTfmSupportBuildWarnings>true</SuppressTfmSupportBuildWarnings>
```

Comment:
We should scope this down to the frameworks that need it. e.g. net8.0/9.0 don't need this.

Reply:
like this: https://github.com/AzureAD/azure-activedirectory-identitymodel-extensions-for-dotnet/pull/3081/files